java-client/idl/recompile_thrift.sh

```diff
@@ -19,21 +19,33 @@
 # under the License.
 #
 
-# method to get thrift 0.11.0:
-#   wget http://mirrors.tuna.tsinghua.edu.cn/apache/thrift/0.11.0/thrift-0.11.0.tar.gz
-#   tar xvf thrift-0.11.0.tar.gz
-#   cd thrift-0.11.0
-#   ./configure --with-csharp=no --with-java=no --with-python=no --with-erlang=no --with-perl=no \
-#       --with-php=no --with-ruby=no --with-haskell=no --with-php_extension=no --with-rs=no --with-go=no
-#   make
-#   sudo make install
-# 
-# Note: by default the thrift will be installed in /usr/local. If you'd like to install it somewhere else,
-# you'd better provide install path with option `--prefix=xxx'.
- 
+function GenThriftTool() {
+    set -e
+    wget --progress=dot:giga https://github.com/apache/thrift/archive/refs/tags/0.11.0.tar.gz -O thrift-0.11.0.tar.gz
```

Comment:
Why we use thrift-0.11.0 in Java and 0.9.3 in all the other places, there must be some historical reason.
I can remember a little there is a compatiable reason in Java client that we shaded thrift 0.11.0, but not sure.
Any body could please clarify it? Many thanks!

Reply:
> Why we use thrift-0.11.0 in Java and 0.9.3 in all the other places, there must be some historical reason. I can remember a little there is a compatiable reason in Java client that we shaded thrift 0.11.0, but not sure. Any body could please clarify it? Many thanks!

I don't know the history's problems., and the problems not just thrift version. One more thing,
in fact, I think the thrift proto files (for interface) should be one shared by cpp and java and others. But the proto files has very large diffenences between cpp and java.